changelog_unreleased/javascript/pr-9711.md

```diff
@@ -0,0 +1,16 @@
+#### Fix comments inside JSX end tag (#9711 by @fisker)
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+<a><// comment
+/a>;
+
+// Prettier stable
+<a></// comment
+a>;
+
+// Prettier master
+<a></ // comment
```

Comment:
That's not how the actual output looks.

Reply:
Thank you, you are right, I first print it like this, then I changed to align with JSXClosingFragment, forgot to update